tools/testsys/src/run.rs

```diff
@@ -151,6 +151,11 @@ struct CliConfig {
     #[arg(long, env = "TESTSYS_INSTANCE_TYPE")]
     instance_type: Option<String>,
 
+    /// Specify the zone that should be used. This is only applicable for aws-* variants.
+    /// It can be omitted for non-aws variants and can be omitted to use default instance types.
+    #[arg(long, env = "TESTSYS_ZONES")]
```

Comment:
Is the intention that we would specify this as TESTSYS_ZONES? I figured this would just be availability zones specific? Can you share how this is intended to be used as an example in the description?